### ISSUE TITLE
Allow a `--skip` option in the `link` command

### ISSUE THREAD

**Issue by mrsimonemms** [cruft/cruft#241]
In my implementations, I'm having to explicitly ignore the `.git` folder (see #198). In the `cruft create` command, there's a `--skip` flag which allows you to ignore paths - this should also be in the `cruft link` command.

This should work in exactly the same way as how it works in the `create` command.

**Comment by justinmayer** [comment]
Agreed, and also adding a `--skip` option to the `update` command (e.g., `cruft update --skip foo.py --skip bar.py`) would be helpful in cases where you already know that conflicts will arise in certain files.
